open_city_profile/settings.py

```diff
@@ -75,6 +80,22 @@
     SECRET_KEY = "xxx"
 
 ALLOWED_HOSTS = env.list("ALLOWED_HOSTS")
+
+if env.str("CSRF_COOKIE_NAME"):
```

Comment:
Note that you don't need to repeat the types here.
So this is enough:
`env("CSRF_COOKIE_NAME")`

Reply:
I changed the conditionals as per your suggestion.